src/styles/style.scss

```diff
@@ -0,0 +1,5 @@
+@import "./variables.scss";
+@import "./page.scss";
+
+@import "./blocks/blocks.scss";
```

Comment:
Consider separating the blocks into individual files following the BEM naming convention, e.g., 'product-card.scss', 'stars.scss', and import them in the 'style.scss' file.